### ISSUE TITLE
Book Carousel Image Height Changes when Tab is Selected

### ISSUE THREAD

**Issue by martypowell** [baltimorecounty/BCPL-assets#253]

**Comment by mxsnyder** [comment]
This would have to be rebuild using either:
- `display: table` and `display: table-cell`
- flexbox

I'll talk to Dan about how bad he wants this.

**Comment by mxsnyder** [comment]
Also, I can't do this with JS to match the heights, since 3/4 of the tabs are hidden.

**Comment by mxsnyder** [comment]
@danfox01 confirmed this isn't worth doing right now.

**Comment by mxsnyder** [comment]
We were using imageTitle instead of title on the third line. This will resolve the issue:

```JavaScript
const imageTitle = $image.attr('title');
const title = encodeURIComponent(imageTitle.split(':')[0]);
const $titleDisplay = $(`<p>${title}</p>`);
```

**Comment by mxsnyder** [comment]
Ready for review on the dev homepage:
https://dev.bcpl.info/

**Comment by martypowell** [comment]
looks AMAZING

**Comment by danfox01** [comment]
A real beauty.

**Comment by danfox01** [comment]
Implemented in 1.4
